draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -82,10 +82,10 @@ sending the close_stream (0x17f7586d2cb570) transport parameter
 understands this transport parameter MUST treat the receipt of a non-empty
 value as a connection error of type TRANSPORT_PARAMETER_ERROR.
 
-In order to allow reliable stream resets in 0-RTT packets, the client MUST
-remember the value of this transport parameter.  If 0-RTT data is accepted by
-the server, the server MUST not disable this extension on the resumed
-connection.
+In order to allow reliable stream resets in 0-RTT packets, both client and
+server MUST remember the value of this transport parameter.  If 0-RTT data is
+accepted by the server, the server MUST not disable this extension on the
```

Comment:
```suggestion
accepted by the server, the server MUST NOT disable this extension on the
```
I know this is from old text, but "MUST not" sounds odd. Maybe it is better to use either "MUST NOT disable" or "MUST enable."

Reply:
I agree. "MUST not" doesn't make sense.